### DISCUSSION TITLE
Is there a reason flameshot would connect to an external IP at any time?

### DISCUSSION THREAD

**Discussion by Veek** [flameshot-org/flameshot#2039] [Q&A]
[https://i.imgur.com/Bpj4Bvu.png](https://i.imgur.com/Bpj4Bvu.png) It seems to be connecting to 13.234.168.60:https (ESTABLISHED)  which seems to be 60.168.234.13.in-addr.arpa. 141 IN      PTR     ec2-13-234-168-60.ap-south-1.compute.amazonaws.com.

**Comment by borgmanJeremy** [comment]
There are two reasons flameshot connects to an IP address 

The first is to query GitHub to see if there is an update. You can disable this if desired in the settings.

The other networking feature is if you upload an image to Imgur.
